API_doc/LibConfig.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+# include "APILibConfig.h"
+
+namespace LibConfig
+{
+
+  /*
+   * manage your subscriptions
+   */
+  struct Subscription
+  {
+  private:
+    onChangeNotifier subscribedFunc;
+  public:
+
+    Subscription(onChangeNotifier);
+
+    void unsubscribe(); // stop your subscription
+  };
+
+  /*
+   * Config class is used to manipulate configuration
+   */
+  struct Config
```

Comment:
This would overload the meaning of `Config`, which beyond a hassle on the implementation side could confuse the user.